scene/gui/item_list.cpp

```diff
@@ -1170,11 +1172,12 @@ void ItemList::_notification(int p_what) {
 					rcache.size.width = width - rcache.position.x;
 				}
 
-				bool should_draw_selected_bg = items[i].selected;
+				bool should_draw_selected_bg = items[i].selected && hovered != i;
+				bool should_draw_hovered_selected_bg = hovered == i && items[i].selected;
```

Comment:
```suggestion
				bool should_draw_hovered_selected_bg = items[i].selected && hovered == i;
```
Why is this reversed?

Reply:
It shouldn't be; it's now corrected.